compiler/verification/z3backend.ml

```diff
@@ -115,7 +115,7 @@ let rec print_z3model_expr (ctx : context) (ty : typ Pos.marked) (e : Expr.expr)
        pretty-printing, we print nothing to remain closer from Catala sources *)
     | TUnit -> ""
     | TInt -> Expr.to_string e
-    | TRat -> failwith "[Z3 model]: Pretty-printing of rational literals not supported"
+    | TRat -> Arithmetic.Real.to_decimal_string e 7
```

Comment:
Instead of `7` you should use the compiler option that is precisely here for this kind of things: https://github.com/CatalaLang/catala/blob/12b31554d673ce905b9cd31bfe89f7b63682a7bd/compiler/utils/cli.ml#L31